flepimop/gempyor_pkg/src/gempyor/utils.py

```diff
@@ -231,14 +278,30 @@ def convert(self, value, view):
 
 @add_method(confuse.ConfigView)
 def as_date(self):
-    "Evaluates an datetime.date or ISO8601 date string, raises ValueError on parsing errors."
+    """
+    Evaluates an datetime.date or ISO8601 date string.
+
+    Args:
+        self: Class instance to convert to date or date string.
+
+    Raises:
+        ValueError: On parsing errors.
```

Comment:
This function does not raise an exception, the exception described here is raised by `ISO8601Date.convert`. The description for that exception should go there not here.